### PR TITLE
More interesting cake transmutation

### PR DESCRIPTION
**Pull request by J0B10**
Starlight transmutation of a Pumpkin should create a pumpkin cheesecake instead of a vanilla cake.

I also used this opportunity to increase the starlight cost of the transmutation by 66.6% to nerv Kekimurus automatic mana farms.

### CODE REVIEW

**Review by J0B10 — COMMENTED** [review]
Starlight transmutation of pumpkins now seems to be bugged.  
In my first tests it worked but now sometimes it doesn't.

I'll have to investigate this, **pls don't merge!**

**Comment by J0B10** [comment]
I managed to spot and resolve this issue by creating items with invalid damage values that get converted to the four pumpkin block states.  

Problem remaining is that these items are also displayed in JEI:

File: scripts/astral.zs [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import crafttweaker.oredict.IOreDict;
 
+//More interesting cake transmutation
+mods.astralsorcery.LightTransmutation.removeTransmutation(<minecraft:cake>, false);
+for meta in 0 to 4 {    
+    //thanks to AstralSorcery's stupid ct integration we must register it for any possible block state 
+    mods.astralsorcery.LightTransmutation.addTransmutation(<minecraft:pumpkin>.withDamage(meta), <harvestcraft:pumpkincheesecake>, 1000.0);
+    //fixme the additional recipes show up in jei with a missing texture
```
Comment (J0B10) [inline scripts/astral.zs:8]: I tried removing the recipes that contain the missing textures from jei:

```suggestion
    val item = <minecraft:pumpkin>.withDamage(meta);
    mods.astralsorcery.LightTransmutation.addTransmutation(item, <harvestcraft:pumpkincheesecake>, 1000.0);
    if (meta > 0) {
        //try to hide the additional states from JEI as they display as missing texture
        mods.jei.JEI.removeAndHide(item);
```

But no changes. Even though `crafttweaker.log` states that the items were removed.

**Comment by J0B10** [comment]
Only ideas left I have is either writing/modifyng a mod to fix this or hide all transmutation recipes from jei using `
mods.jei.JEI.hideCategory("astralsorcery.lightTransmutation");`

**Comment by sebinside** [comment]
RIP

**Pull request closed without merging**